### DISCUSSION TITLE
pluck operator is a standard operator it is also available in Lodash and Ramda - It belongs into the library!

### DISCUSSION THREAD

**Discussion by hansschenker** [ReactiveX/rxjs#7479] [General]
### Describe the bug

pluck operator is a standard operator it is also available in Lodash and Ramda - It belongs into the library!

### Expected behavior

pluck operator is a standard operator it is also available in Lodash and Ramda - It belongs into the library!

### Reproduction code

```typescript
please do not leave out pluck operator!
```

### Version

version 8 to n

**Comment by AlexMoutonNoble** [comment]
Use map to lift the lodash operator into an observable.

**Reply by hansschenker** [reply]
I do not want to use another library if I have the solution in Rxjs already. Pluck operator does not raise the memory footprint of Rxjs but raises the DX footprint to a comfortable level!
